kvmagent/kvmagent/test/utils/vm_utils.py

```
@@ -280,6 +292,14 @@ def build_virtio_vol_with_iothreadpin(vol_uuid, vol_path, iothread_id, pin):
     return data_vol, {"ioThreadId": iothread_id, "pin": pin, "volumeUuid": vol_uuid}
 
 
+def build_virtio_vol_with_iothreadpin(vol_uuid, vol_path):
```

Ruff (0.12.2)

Redefinition of unused `build_virtio_vol_with_iothreadpin` from line 285

(F811)